lib/cictl/common.rb

```diff
@@ -31,14 +31,16 @@ def solr_client
 
     # Coerce a user-supplied Date or String to a Date within a block.
     def with_date(obj)
-      date = obj
-      unless obj.is_a? Date
+      date = if obj.respond_to?(:to_date)
```

Comment:
`DateTime.is_a? Date` is true, but that also includes some spurious time zone information that is unlikely to be helpful and could end up with bad math in some cases... In this case we really want a `Date` if we can; `DateTime#to_date`, `Time#to_date`, and `Date#to_date` will all do that.